indent/javascript.vim

```diff
@@ -240,7 +240,7 @@ function s:LineHasOpeningBrackets(lnum)
     endif
     let pos = match(line, '[][(){}]', pos + 1)
   endwhile
-  return (open_0 > 0) . (open_2 > 0) . (open_4 > 0)
+  return (open_0 > 0?1:(open_0 == 0 ? 0 : 2)) . (open_2 > 0) . (open_4 > 0)
```

Comment:
Could you explain what this does?

Let's include a space either side of `?` and `:`.